### PR TITLE
Add attributions for weather status in modal

### PR DESCRIPTION
**Pull request by julien-nc**
Not sure about the header text and the attributions themselves :grin:.

### CODE REVIEW

**Comment by jancborchardt** [comment]
I’d say let’s use "Weather service" as a heading, and there we could also put a checkbox saying "Enable weather" which by default is checked, but could be unchecked if people don’ŧ want weather at all.

What do you think @eneiluj @juliushaertl?

**Comment by eneiluj** [comment]
@jancborchardt Title changed, thanks.
Yes the checkbox next to the title makes sense. I might do it tomorrow.

**Comment by eneiluj** [comment]
@jancborchardt @juliushaertl I was too lazy to create another branch+PR for this: the order of the statuses is now inverted (alphabetic sort by name for the moment).

I started to make it possible to toggle weather status, i can get rid of it by removing it from the `registeredStatus` array. Then it was the end of the day :grin: and i didn't find out how to add it and render it again. @juliushaertl if you point the right direction i can do it.

cheers

**Comment by jancborchardt** [comment]
Adjusted the design :)
image

File: apps/dashboard/src/App.vue [hunk 1]
```diff
@@ -59,6 +59,14 @@
 
 				<h3>{{ t('dashboard', 'Change background image') }}</h3>
 				<BackgroundSettings :background="background" @update:background="updateBackground" />
+
+				<h3>{{ t('dashboard', 'Weather service') }}</h3>
```
Comment (juliushaertl) [inline apps/dashboard/src/App.vue:63]: Ideally we should put this in the weather_status app somehow, but fine for now I'd say :+1:

**Review by juliusknorr — APPROVED** [review]

**Review by jancborchardt — APPROVED** [review]

**Pull request merged**